src/ZiraLink.Domain/Enums/RowState.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace ZiraLink.Domain.Enums
 {
-    public enum ProjectState
+    public enum RowState
```

Comment:
I wanted to use this enum everywhere. Even in then new features which we may have.

Reply:
The flag is specifically intended for indicating the status of the project, rather than the status of the row.

A project typically has a status that indicates whether it is active or inactive. In addition, it may also have a row status that denotes, for instance, if it has been removed or not.

Reply:
I got it. Thank you so much.